contracts/Payroll.sol

```diff
@@ -325,7 +325,7 @@ contract Payroll is Initializable, OwnableUpgradeable, ReentrancyGuardUpgradeabl
      * Perform the payments to the given addresses and amounts, public method.
      * @param _payments The array of the Payment data.
      */
-    function performMultiPayment(Payment[] calldata _payments) external nonReentrant {
+    function performMultiPayment(Payment[] calldata _payments) external payable nonReentrant {
```

Comment:
`performSwapV2AndPayment ` should also be changed to be payable, same for V3, otherwise the _performMultiPayment won't work as expected.
I knwo that swap with native token is not able yet on those, but the transfer should be